### PR TITLE
Base/Theme: Added body/main background colour variables.

### PR DESCRIPTION
**Pull request by EricDunsworth**

### CODE REVIEW

File: theme/_variables.scss [hunk 1]
```diff
@@ -1,8 +1,14 @@
 // Theme Variable overrides
 
+// Body
+$body-bg: #f9f9f9; // Body element's background colour (Bootstrap override).
```
Comment (LaurentGoderre) [inline theme/_variables.scss]: Where is this called?

Comment (EricDunsworth) [inline theme/_variables.scss]: @LaurentGoderre In the version of Bootstrap WET currently uses (3.3.1)...
* It's declared here:
  * https://github.com/twbs/bootstrap-sass/blob/93f42bc913bdd4c9df3d4652bd5168b4ede870c6/assets/stylesheets/bootstrap/_variables.scss#L34
* And used here:
  * https://github.com/twbs/bootstrap-sass/blob/93f42bc913bdd4c9df3d4652bd5168b4ede870c6/assets/stylesheets/bootstrap/_scaffolding.scss#L32
* Its value is also re-assigned to some other Bootstrap variables (neither WET nor GCWeb currently re-use them or their parent CSS classes):
  * https://github.com/twbs/bootstrap-sass/blob/93f42bc913bdd4c9df3d4652bd5168b4ede870c6/assets/stylesheets/bootstrap/_variables.scss#L431
  * https://github.com/twbs/bootstrap-sass/blob/93f42bc913bdd4c9df3d4652bd5168b4ede870c6/assets/stylesheets/bootstrap/_variables.scss#L436
  * https://github.com/twbs/bootstrap-sass/blob/93f42bc913bdd4c9df3d4652bd5168b4ede870c6/assets/stylesheets/bootstrap/_variables.scss#L436

Comment (LaurentGoderre) [inline theme/_variables.scss]: Oh yeah....duh. Sorry about that!

**Comment by EricDunsworth** [comment]
FYI I've updated this PR to set $main-bg to null (instead of transparent) by default and added an if condition to only use it contains a real value. That way, themes that don't use custom background colours for their ``<main>`` elements won't have to deal with pointless extra CSS..

Btw I'll go ahead and create similar PRs to wet-boew/GCWeb#1220 for the other theme repos.

**Review by LaurentGoderre — APPROVED** [review]
LGTM!

**Pull request merged**